src/commands/build-and-push-image.yml

```diff
@@ -30,6 +30,28 @@ parameters:
       Enable Docker layer caching if using remote Docker engine.
       Defaults to false.
 
+  docker-login:
+    type: boolean
+    default: false
+    description: >
+      Enable dockerhub authentication. Defaults to false.
+
+  dockerhub-username:
+    type: string
+    default: DOCKERHUB_USERNAME
+    description: >
+      Dockerhub username to be configured. Set this to the name of
+      the environment variable you will set to hold this
+      value, i.e. DOCKERHUB_USERNAME.
+
+  dockerhub-password:
+    type: string
+    default: DOCKERHUB_PASSWORD
+    description: >
+      Dockerhub password to be configured. Set this to the name of
+      the environment variable you will set to hold this
+      value, i.e. DOCKERHUB_PASSWORD.
```

Comment:
These should probably be `env_var_name` types.

Reply:
You're right. Good catch. It's updated: https://github.com/CircleCI-Public/aws-ecr-orb/pull/120/commits/ef3bdbaab138bc0d8fc4a9abc082caaea35756b5